aws/logs_monitoring/README.md

```diff
@@ -365,6 +365,10 @@ To test different patterns against your logs, turn on [debug logs](#troubleshoot
 
 `AdditionalTargetLambdaARNs`
 : Comma separated list of Lambda ARNs that will get called asynchronously with the same `event` the Datadog Forwarder receives.
+`InstallFromLayer`
+: Disable to use legacy installation flow, (via Github), for regions where Lambda layers aren't supported, such as regions in China. Defaults to true.
```

Comment:
I would reword this, I don't think it's obvious what `legacy installation flow, (via Github)` entails.

Reply:
Also, I think "regions where Lambda layers aren't supported, such as regions in China" is a bit misleading. It looks like Lambda layers are supported in China, but we just don't publish our layers there.